src/public/device.ts

```diff
@@ -0,0 +1,93 @@
+import { GlobalVars } from '../internal/globalVars';
+import { ensureInitialized, sendMessageRequestToParent } from '../internal/internalAPIs';
+import { frameContexts } from '../internal/constants';
+
+/**
+ * Namespace for device related APIs. These are implemented only for mobile (Android and iOS)
+ * On desktop, these APIs will be a noop.
```

Comment:
Shouldn't JS SDK return error if webapp invokes it for Desktop platform ?

Reply:
Yes. I will check the behavior and do the needful.